### ISSUE TITLE
Team drive support

### ISSUE THREAD

**Issue by vijaygopal1234** [dsoprea/GDriveFS#184]
I think this would be very useful, however I'm not sure how hard it would be.

**Comment by dsoprea** [comment]
I'm not sure it isn't. Other than some properties not being populated for objects owned by a Team, it seems like the root-level Team folders should be present (if your account is associated with them). Have you tried?

**Comment by vijaygopal1234** [comment]
I have tested with an account that was shared a Team drive. I could only see my personal drive when I mounted.

**Comment by dsoprea** [comment]
You're right. It comes in alongside V3 changes though does not appear to require them.

Implementation roadmap:

https://developers.google.com/drive/v3/web/enable-teamdrives

**Comment by vijaygopal1234** [comment]
Oh I see, well I hope you have the time to implement them. Thanks for GDriveFS!
